test/wrappers_test.js

```diff
@@ -146,3 +149,75 @@ describe('transform-node', function() {
     chai.expect(spy).to.have.been.calledWith();
   });
 });
+
+describe('output-wrapper', function() {
+  let output;
+
+  beforeEach(() => {
+    fs.mkdirSync('tmp');
```

Comment:
Please avoid making tmp dirs in project, use something like the `tmp` package instead